docs/examples/contact-manager.md

```diff
@@ -0,0 +1,215 @@
+connection = sqlite3.connect('sqllite.db')
+
+# Create a cursor object to execute SQL commands
+cursor = connection.cursor()
+
+# Create a table
+cursor.execute('''CREATE TABLE IF NOT EXISTS contacts (
+                    phoneNumber TEXT PRIMARY KEY,
+                    name TEXT NOT NULL,
+                    surname TEXT NOT NULL,
+                    email TEXT
+                )''')
+
+
+# Commit the transaction
+connection.commit()
+
+
+# Close cursor and connection
+cursor.close()
+connection.close()
+```
+
+With our table and database file we are set for the next section!
+
+#### Adding to the contact database
+
+Now we will be making a function which will be responsible for inserting the contact's data into the database. We will get the info from the database with `c.execute("SELECT * FROM contacts WHERE phoneNumber = ?", (phoneNumber,)")` and we will inserting it with `c.execute("INSERT INTO contacts VALUES (?)", (params,))`, params will equal contact's phone number, name, surname and email. Here is what it should look like:
+
+```python
+class Contact(Skill):
+ @match_regex(r"Add contact: (\d+), (.+), (.+), (\S+@\S+)") #phoneNumber, name, surname, email
+ async def addContact(self, message):
+    phoneNumber = message.regex.group(1)  # Extract the phoneNumber
+    name = message.regex.group(2) 
+    surname = message.regex.group(3) 
+    email =  message.regex.group(4) 
+    c.execute("SELECT * FROM contacts WHERE phoneNumber = ?", (phoneNumber,))
+    existing_contact = c.fetchone()
+    if existing_contact:
+     await message.respond(f"Contact already exists!")
+    else:
+      c.execute("INSERT INTO contacts (phoneNumber, name, surname, email) VALUES (?, ?, ?, ?)", (phoneNumber, name, surname, email))
+      conn.commit()
+    await message.respond(f"Contact with Phone Number {phoneNumber} added to the Contact Book!")
+```
+We prevent duplicating one contact into the database by executing this:
+```python
+c.execute("SELECT * FROM contacts WHERE phoneNumber = ?", (phoneNumber,)) 
+existing_contact = c.fetchone()
+```
+### Getting The Data
+
+This part involves listing the contacts that have been added to the database. We will be doing this by getting the data from the database the same as how we did for the adding to the database which is `c.execute("SELECT * FROM contacts")`. With the data we can use a for loop to go through all the things in the database. It should look something like this:
+
+```python
+@match_regex(r"Show contacts")     
+ async def showContacts(self, message):
+  c.execute("SELECT * FROM contacts")
+  rows = c.fetchall()
+  if len(rows) == 0:
+    await message.respond('No Contacts added, Please Use Command "Add contact: PhoneNumber, Name, Surname, Email"')
+  else: 
+     for row in rows:
+        phoneNumber, name, surname, email = row
+        await message.respond(f'Phone Number: {phoneNumber}, Name: {name}, Surname: {surname} and Email: {email}')
```

Comment:
Hmm indentation looks a bit broken here too. Could you check all the codeblocks?